notebooks/book2/10/mc_estimate_pi.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.            plt.figure()
</pre><p>What is use of this line? We can put this line out of conditions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I have fixed the issues that you have mentioned.</p>